pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -3462,6 +3454,16 @@ public void testGroupByAggregationWithLimitZero(boolean useMultiStageQueryEngine
     testQuery("SELECT Origin, SUM(ArrDelay) FROM mytable GROUP BY Origin LIMIT 0");
   }
 
+  @Test(dataProvider = "useBothQueryEngines")
+  public void testFilteredAggregationWithGroupByOrdering(boolean useMultiStageQueryEngine)
+    throws Exception {
+    setUseMultiStageQueryEngine(useMultiStageQueryEngine);
+
+    // Test the ordering is correctly applied to the correct aggregation (the one without FILTER clause)
```

Comment:
nit: I would just add a mention to #13749 in this comment so readers can have more context.

We don't usually do that and it has been useful to me when reading other open source projects I know less about.